esp-config/src/generate/mod.rs

```diff
@@ -51,6 +52,10 @@ impl fmt::Display for Error {
 /// that match the given prefix. It will then attempt to parse the [`Value`] and
 /// run any validators which have been specified.
 ///
+/// [`Stability::Unstable`] features will only be enabled if the `unstable`
```

Comment:
Can we get esp config to generate this text for the documentation for other packages? Right now the bare table with just the "Unstable" marker doesn't mean much to the user. Maybe we should just add this to the docs of the other packages and move on though :thinking:

Reply:
My thinking was that those packages are unstable as they are, so there's no need to add an unstable feature for them just yet.

Reply:
Yeah fair enough, as long as esp-hal is documented we can punt this down the road.